test/e2e/storage/drivers/in_tree.go

```diff
@@ -144,11 +146,14 @@ func (n *nfsDriver) GetClaimSize() string {
 }
 
 func (n *nfsDriver) CreateDriver() {
-	f := n.driverInfo.Framework
+	f := n.driverInfo.Config.Framework
 	cs := f.ClientSet
 	ns := f.Namespace
 	n.externalPluginName = fmt.Sprintf("example.com/nfs-%s", ns.Name)
 
+	// Reset config. It might have been modified by a previous CreateVolume call.
```

Comment:
Tests can run in parallel. Does each test case get it's own unique object, or share the same object?  If each test case has a new object, then this is not needed.  If they share the same object, then this may not work very well.

Reply:
Opened up https://github.com/kubernetes/kubernetes/issues/72288 to consider refactoring this later.  I don't think this is a clean abstraction